### ISSUE TITLE
Breadcrumbs/Link component doesn't meet WCAG 2 AA contrast ratio thresholds in the High Contrast Black theme.

### ISSUE THREAD

**Issue by mdasari84** [SAP/ui5-webcomponents#5195]
### **Bug Description**
Breadcrumbs/Link component doesn't meet WCAG 2 AA contrast ratio thresholds in the High Contrast Black theme.

### **Expected Behavior**
Links should have a contrast ratio of 4.5:1

### **Steps to Reproduce**
1. Go to https://sap.github.io/ui5-webcomponents/playground/components/Breadcrumbs/
2. Select the settings icon, change the theme field to "High Contrast Black", and click Apply button
3. Check the links in the Breadcrumbs/ Link
4. Please refer to the attached screenshots.

### **Context**
- UI5 Web Components version: 1.3.x
- OS/Platform: Windows
- Browser: Chrome
- Affected component: Breadcrumbs, Link

### **Log Output / Stack Trace / Screenshots**
```
![Breadcrumbs component](https://user-images.githubusercontent.com/84945056/167726251-349f8475-5db4-443b-ab74-123023006ff6.png)
![Link component](https://user-images.githubusercontent.com/84945056/167726254-056bdc6d-40d6-447f-b77b-0fd9247c7b99.png)

```
### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: SuccessFactors.

**Comment by niyap** [comment]
Hello Colleagues,

When the "href" property of the ui5-link has "#" as a value, the browser's native style overwrites the one specified by you and the link's color is not correct in HC themes and that causes the accessibility issue.

Open the test page: https://sap.github.io/ui5-webcomponents/master/playground/main/pages/Link/?sap-ui-theme=sap_fiori_3_hcb
Change the "href" attribute's value to "#" and notice the color of the link -  it is blue. Tested with the Color Contast Checker and the contrast ratio does not specified the accessibility requirements.

Could you please look over?

Kind Regards,
Niya
